### ISSUE TITLE
Align on accessibility's role in content audit

### ISSUE THREAD

**Issue by sarah-sch** [uswds/uswds#5203]
Meet with Amy C. and Bonnie to decide what should be reported in accessibility column of content audit. Current working plan is to make note of only very high-level concerns. We don't know the future of each page at this point and don't want to get too granular.

**Comment by sarah-sch** [comment]
Meeting scheduled 3/31

**Comment by sarah-sch** [comment]
Met with Amy C. 4/29 and added notes to [planning document](https://docs.google.com/document/d/1AT84ghbbr479B1-uMXoyVam4UJiOCrsLDALpXW5nlaU/edit?usp=sharing). 

Will continue discussion with Bonnie at our 4/3 checkin.

**Comment by sarah-sch** [comment]
Closing this ticket as done. Next stop will be filling out 5-10 audit rows for approval by @amycole501 before diving into full audit.
